### PR TITLE
Added json ld info

### PR DESCRIPTION
**Pull request by gopalshimpi**
## Why?
* Added Jsonld Information for product details page.
## This change Addresses the need by:
*  Product details page can have Jsonld information for seo.

### CODE REVIEW

File: src/app/product/components/product-detail-page/product-details/product-details.component.ts [hunk 1]
```diff
@@ -183,7 +183,7 @@ export class ProductDetailsComponent implements OnInit {
       'itemCondition': 'https://schema.org/NewCondition',
       'brand': {
         '@type': 'Thing',
-        'name': `${this.brand.name}`
+        'name': `AviaCommerce`
```
Comment (pkrawat1) [inline src/app/product/components/product-detail-page/product-details/product-details.component.ts]: Why changed ?

Comment (gopalshimpi) [reply src/app/product/components/product-detail-page/product-details/product-details.component.ts]: Ok

**Comment by gopalshimpi** [comment]
Created new pull request to avoid merge commits.

**Comment by gopalshimpi** [comment]
Created new pull request to avoid merge commits.

**Pull request closed without merging**